frontend/src/features/frame/components/topbar/TopBar.tsx

```diff
@@ -140,14 +152,41 @@ const TopBar = () => {
           padding: '4px !important',
         }}
       >
-        <Grid container sx={{ width: '100%' }}>
-          <Logo />
+        <Grid container sx={{ width: '100%', maxWidth: '100%' }}>
           <Hidden mdDown>
+            <Logo />
             <Grid item md={4}>
               {options?.withSearchBar && <Search />}
             </Grid>
+            <AccountInfo />
+          </Hidden>
+          <Hidden mdUp>
```

Comment:
Code Style:

Because the `mdDown` and `mdUp` are now separated, maybe it would be useful to create new components `MobileTopBar` and `LargeScreenTobBar` ?

Then
```
const TopBar = () => {
  [...]

  return (
        [...]
        <Grid container sx={{ width: '100%', maxWidth: '100%' }}>
          <Hidden mdDown>
            <LargeScreenTopBar />
          </Hidden>
          <Hidden mdUp>
            <MobileTopBar />
          </Hidden>
        </Grid>
        [...]
    )
```